mycroft/skills/skill_manager.py

```diff
@@ -115,19 +115,27 @@ def _shutdown_skill(instance):
 
 
 def _get_skill_locations(conf=None):
-    # next load all valid XDG paths
+    # the contents of each skills directory must be individual skill folders
+    # we are still dependent on the mycroft-core structure of skill_id/__init__.py
+
+    conf = conf or Configuration.get(remote=False)
+    # load all valid XDG paths
     # NOTE: skills are actually code, but treated as user data!
-    # they should be considered applets rather than applications
+    # they should be considered applets rather than full applications
     xdg_paths = list(reversed(
         [os.path.join(p, "skills")
          for p in xdg.BaseDirectory.load_data_paths(BASE_FOLDER)]
     ))
 
-    # the highest priority folder to load is the old hardcoded
-    # /opt/mycroft/skills folder
+    # load the old hardcoded /opt/mycroft/skills folder
+    # needed for integration with msm, assume external tools will install skills here!
     default = get_skills_directory(conf)
     if default not in xdg_paths:
         xdg_paths.append(default)
+
+    # load additional explicitly configured directories
```

Comment:
Consider noting that `extra_directories` is expected to be a directory containing skill sibdirectories, rather than a list of skill folders